creator-node/src/routes/files.js

```diff
@@ -119,16 +120,16 @@ module.exports = function (app) {
       await t.rollback()
       return errorResponseServerError(e)
     }
-  }))
+  }), postMiddleware)
 
   /** upload metadata to IPFS and save in Files table */
-  app.post('/metadata', authMiddleware, nodeSyncMiddleware, handleResponse(async (req, res) => {
+  app.post('/metadata', authMiddleware, preMiddleware, nodeSyncMiddleware, handleResponse(async (req, res) => {
```

Comment:
I think this doesn't need pre-middleware

Reply:
I removed it, just summarizing:
- We want to upload the metadata to the creator node *before* we write to chain the new CID for the user's metadata (because the CID could very well not exist if we did it the other way around)
- The metadata isn't really associated with the user yet until we call associate, so it's ok that we don't check if it's the primary here